packages/react-instantsearch-dom/src/widgets/Stats.ts

```diff
@@ -2,12 +2,12 @@ import { connectStats } from 'react-instantsearch-core';
 import Stats from '../components/Stats';
 
 /**
- * The Stats component displays the total number of matching hits and the time it took to get them (time spent in the Algolia server).
+ * The Stats component displays the total number of matching hits and the time it took to get them (time spent in the Algolia server) if `relevancyStrictness` is `undefined`. Displays the total number of matching sorted hits out of the matching hits and the time it took to get them (time spent in the Algolia server) otherwise.
  * @name Stats
  * @kind widget
  * @themeKey ais-Stats - the root div of the widget
  * @themeKey ais-Stats-text - the text of the widget - the count of items for each item
- * @translationkey stats - The string displayed by the stats widget. You get function(n, ms) and you need to return a string. n is a number of hits retrieved, ms is a processed time.
+ * @translationkey stats - The string displayed by the stats widget. You get function({ n, nSorted, isSmartSorted, ms }) and you need to return a string. `n` is a number of hits retrieved, `nSorted` is a number of sorted hits retrieved, `isSmartSorted` is a boolean translating a `relevancyStrictness` set between 0 and 100, `ms` is a processed time.
```

Comment:
I'm not sure if this change will be accepted but I feel like it's better to have an object to avoid retrieving unused params

Reply:
unfortunately that's a breaking change. Can you make the signature `(n, ms, {nbHits, nbSortedHits, isSmartSorted})` to avoid the breakage?

Reply:
Sure. As `n` is the `nbHits`, `{nbSortedHits, isSmartSorted}` should be enough

Reply:
I put nbHits in the third argument so you can use only that one, but on reflection, a regular argument list might be most consistent: `(nbHits, ms, nbSortedHits, isSmartSorted)`, even though it's odd that ms is in the middle

Reply:
Got it!

Reply:
Done in https://github.com/algolia/react-instantsearch/pull/3010/commits/b1886239778a3be9f1c5ff2bed5c57030de0fe32